hack/bundle-automation/config.yaml

```diff
@@ -6,9 +6,7 @@
   name: hive-operator
   imageMappings:
     hive: openshift_hive
-    stolostron: openshift_hive
-  imageRepoOverride: stolostron
-  imageTagOverride: 2.4-local-30c82d3f1a0ef1edaedcbed9d92b840732ad2e75
+    disable-image-validation: openshift_hive
```

Comment:
I'm not following why the disable-image-validation property is being added to the config.  I don't see it being used in any way by other parts of this change.  The fix is hive specific and we're accomplishing it by adding a dummy incantation of  their tool's --image-repo option to the hive-specific gen script, so no need for this to be controlled at the config level.  I'd suggest removing this property  from the confog.

Reply:
@joeg-pro I added the `disable-image-validation` property to the `config.yaml` file to resolve the error that would be  thrown without it: 

```bash
INFO:root:Adding Overrides (set --skipOverrides=true to skip) ...
INFO:root:Updating Helm chart 'pkg/templates/charts/toggle/hive-operator' with onboarding requirements ...
INFO:root:Fixing image and pull policy references in deployments and values.yaml ...
CRITICAL:root:No image key mapping provided for imageKey: disable-image-validation
make: *** [regenerate-operator-bundles] Error 1
```

Since there is no `imageKey`, the job will fail before the bundle can be regenerated properly.

Reply:
Sorry I didn't see this earlier.  I see that I was misinterpreting the disable-image-validation thing in config as some new control property, vs. being just a new image-key mapping.  Sorry about that.  Seeing what is going on, we can avoid the new mapping by using a better version of the hack in gen-hive-bundle.sh.  (I had not realized that the hack would result in the CSV having a image ref that we couldn't recognize.)

I've pushed an update that eliminates the extra key, and still works I think.